appveyor.yml

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: 0.15.0-SNAPSHOT.{build}-{branch}
+
+shallow_clone: true
+
+platform: x64
+configuration: Debug
```

Comment:
I would prefer to start with one setup. During testing I think I noticed that these parameters seem to not propagate to build scripts, that's why I had to pass them explicitly to build and test command. Later, when our local runtime tests pass consistently, and we start work on adding Yarn and HDInsight tests, we can also look into adding release configuration.